smb_navigation/launch/navigate2d_cmu.launch

```diff
@@ -0,0 +1,102 @@
+  
+    <!-- Remapping State Estimation Topic -->
+    <remap from="/state_estimation" to="$(arg state_estimation_topic)"/>
+    <remap from="/registered_scan" to="$(arg registered_scan_topic)"/> 
+  
+    <!-- Pointcloud Transformer Node -->
+    <node name="pointcloud_transformer" pkg="smb_navigation" type="pointcloud_transformer_node" output="screen">
+        <param name="global_frame"     value="$(arg global_frame)"/>
+        <param name="pcd_topic"        value="$(arg pcd_topic)"/>
+        <param name="transformed_pcd_topic"        value="$(arg transformed_pcd_topic)"/>
+    </node>
+
+    <!-- Odometry Transformer Node -->
+    <group if="$(arg use_msf)">
+      <node name="odometry_transformer" pkg="smb_navigation" type="odometry_transformer_node" output="screen">
+        <param name="target_frame"     value="$(arg sensor_frame)"/>
+        <param name="odom_topic"        value="/graph_msf/est_odometry_world_imu"/>
+        <param name="transformed_odom_topic"        value="$(arg transformed_odom_topic)"/>
+      </node>
+    </group>
+    
+  
+    <!-- Static Transform adapt to CMU hardcoded frame -->
+    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="attach_map_link" args="0 0 0 0 0 0 $(arg global_frame) map" /> -->
+    <node pkg="tf2_ros" type="static_transform_publisher" name="attach_sensor_link" args="0 0 0 0 0 0 $(arg sensor_frame) sensor" />
+
+    <!-- Base CMU exploration layer-->  
+    <include file="$(find local_planner)/launch/local_planner.launch" >
+      <arg name="dirDiffThre" value="$(arg dirDiffThre)"/>
+      <arg name="cameraOffsetZ" value="$(arg cameraOffsetZ)"/>
+      <arg name="maxSpeed" value="$(arg maxSpeed)"/>
+      <arg name="maxYawRate" value="$(arg maxYawRate)"/>
+      <arg name="maxAccel" value="$(arg maxAccel)" />
+      <arg name="goalX" value="$(arg vehicleX)"/>
+      <arg name="goalY" value="$(arg vehicleY)"/>
+    </include>
+  
+    <include file="$(find terrain_analysis)/launch/terrain_analysis.launch">
+      <arg name="world_frame"     value="$(arg global_frame)"/>
+      <arg name="vehicleHeight" value="$(arg vehicleHeight)"/>
+    </include>
+
+    <include file="$(find terrain_analysis_ext)/launch/terrain_analysis_ext.launch" >
+      <arg name="world_frame"     value="$(arg global_frame)"/>
+      <arg name="checkTerrainConn" value="$(arg checkTerrainConn)"/>
+      <arg name="vehicleHeight" value="$(arg vehicleHeight)"/>
+    </include>
+  
+    <include file="$(find sensor_scan_generation)/launch/sensor_scan_generation.launch">
+      <arg name="world_frame"     value="$(arg global_frame)"/>
+    </include>
+    
+    <!-- <include file="$(find visualization_tools)/launch/visualization_tools.launch" >
+      <arg name="world_name" value="$(arg world_name)"/>
+    </include> -->
+
+    <!-- FAR Planner -->
+    <group unless="$(arg exploration)">
+        <group if="$(arg launch_far_planner)">
```

Comment:
is it necessary to have two groups here? or could one of the arguments be left out?

Reply:
kept only launch_far_planner, will sync to doc